### PR TITLE
MathJax Directive

### LINKED ISSUE #6: LEOO: redirect route
Add into the app-routing:
`{ 
        // path to specific component for the BEOL project
        path: 'leoo/:rn',
        component: LeooComponent
    }`

- [x] implement the creation of a query that gets the Iri of the LEOO letter (cf Salsah leoo.component.ts )
- [ ] test it

### LINKED ISSUE #8: MathJax: fix internal links
- [ ] adapt @HostListener for internal links

### LINKED ISSUE #10: MathJax: load it when clicking on one specific element

### PR DESCRIPTION
**Pull request by tobiasschweizer**
- re-implements MathJax directive for BEOL resources
- adds HostListeners to HTML received from Knora
- display a linked internal resource (click  and hovering events) in a text value
- puts routing by resource type in a central place
- only render math when necessary 
- reactivate LEOO route (resolve Repertorium number)

fixes #10 
fixes #8 
fixes #6

### CODE REVIEW

File: src/app/resource/letter/letter.component.ts [hunk 1]
```diff
@@ -182,12 +182,12 @@ export class LetterComponent implements OnDestroy, OnInit {
         });
 
         // subscribe to the router events to reload the content
-        this.navigationSubscription = this._router.events.subscribe((e: any) => {
+        /*this.navigationSubscription = this._router.events.subscribe((e: any) => {
```
Comment (tobiasschweizer) [inline src/app/resource/letter/letter.component.ts]: @flavens @kilchenmann I do not understand what this is. But it caused troubles: resources were loaded twice

File: src/app/resource/resource.component.ts [hunk 2]
```diff
@@ -131,12 +131,12 @@ export class ResourceComponent implements OnChanges, OnDestroy, OnInit {
     });
 
     // subscribe to the router events
-    this.navigationSubscription = this._router.events.subscribe((e: any) => {
+    /* this.navigationSubscription = this._router.events.subscribe((e: any) => {
```
Comment (tobiasschweizer) [inline src/app/resource/resource.component.ts]: see 
https://github.com/dhlab-basel/beol/pull/13/files#r225964400

**Comment by tobiasschweizer** [comment]
I identified some problems:

- when clicking on an internal link (standoff link), the URL changes but the resource is not reloaded. I guess this is because I commented out the `NavigationEnd`thing. The problem was that with `NavigationEnd`, the same resource was loaded twice.
- sometimes the scroll event triggers immediately and fetches the next page of results, also when the user does nothing

**Comment by tobiasschweizer** [comment]
> sometimes the scroll event triggers immediately and fetches the next page of results, also when the user does nothing

I think this depends on the position of the mouse. This might trigger the scroll event. Whenever two extended searches are performed at the same time, the same ontology information is requested several times from Knora: the first ontology request is not done yet, and therefore a second one is performed, requesting the same information.

**Review by flavens — APPROVED** [review]

**Review by flavens — APPROVED** [review]

**Review by flavens — APPROVED** [review]

**Review by flavens — APPROVED** [review]

**Review by flavens — APPROVED** [review]

**Review by flavens — APPROVED** [review]

**Review by flavens — APPROVED** [review]

**Pull request merged**